Fix RSS trigger failure issue in Logic Apps Quickstart guide (#126368)
Azure/azure-content: articles/logic-apps/quickstart-create-example-consumption-workflow.md

   :::image type="content" source="media/quickstart-create-example-consumption-workflow/add-rss-trigger-settings.png" alt-text="Screenshot shows the RSS trigger settings, including RSS URL, frequency, interval, and others." lightbox="media/quickstart-create-example-consumption-workflow/add-rss-trigger-settings.png":::

1. On the designer toolbar, select **Code view**.

1. In the code editor, find the line **`"feedUrl": "@{encodeURIComponent(encodeURIComponent(`https://feeds.a.dj.com/rss/RSSMarketsMain.xml'))}"`**.

1. Remove the extra function named **`encodeURIComponent()`** so that you have only one instance, for example: 
   
   **`"feedUrl": "@{encodeURIComponent('https://feeds.a.dj.com/rss/RSSMarketsMain.xml')}"`**  

   This change is necessary to remove double-encoding behavior, which requires manual correction.  
   
1. Switch back to the designer.

1. Save your workflow. On the designer toolbar, select **Save**.

> 

Reviewer: Remove extra blank line.

```suggestion
```

1. Save your workflow. On the designer toolbar, select **Save**.

> 1. On the designer toolbar, select **Code view**. In the code editor, change the line **`"feedUrl": "@{encodeURIComponent(encodeURIComponent(`https://feeds.a.dj.com/rss/RSSMarketsMain.xml'))}"`** to the following version:  

Reviewer: Hi @LimitlessDevv, 

Could you please move all the new content to appear *before* the last step for saving your workflow. That way, the last step should be to save your workflow.

I also included some helper text to summarize the change. Thanks!

```suggestion
1. On the designer toolbar, select **Code view**.

1. In the code editor, find the line **`"feedUrl": "@{encodeURIComponent(encodeURIComponent(`https://feeds.a.dj.com/rss/RSSMarketsMain.xml'))}"`**.

1. Remove the extra function named **`encodeURIComponent()`** so that you have only one instance, for example: 
```

> 1. Save your workflow. On the designer toolbar, select **Save**.

Reviewer: Move this step to the end.

```suggestion
```

> 

Reviewer: Delete extra blank line.

```suggestion
```

1. On the designer toolbar, select **Code view**.

1. In the code editor, find the line **`"feedUrl": "@{encodeURIComponent(encodeURIComponent(`https://feeds.a.dj.com/rss/RSSMarketsMain.xml'))}"`**.

1. Remove the extra function named **`encodeURIComponent()`** so that you have only one instance, for example: 
   
   **`"feedUrl": "@{encodeURIComponent('https://feeds.a.dj.com/rss/RSSMarketsMain.xml')}"`**  

   This change is necessary to remove double-encoding behavior, which requires manual correction.  
   

> 1. Switch back to the designer.  

Reviewer: Put the save workflow step at the end.

```suggestion
1. Switch back to the designer.

1. Save your workflow. On the designer toolbar, select **Save**.

```

   This step instantly publishes your logic app resource and workflow live in the Azure portal. However, the trigger only checks the RSS feed without taking any other actions. So, you need to add an action to specify what you want to happen when the trigger fires.

> 1. On the logic app code view,  

Reviewer: @LimitlessDevv: Thanks for your contribution! I'd move this addition before the last step to save the workflow.

```suggestion
1. On the designer toolbar, select **Code view**. In the code editor, change the line **`"feedUrl": "@{encodeURIComponent(encodeURIComponent(`https://feeds.a.dj.com/rss/RSSMarketsMain.xml'))}"`** to the following version:

   **`"feedUrl": "@{encodeURIComponent('https://feeds.a.dj.com/rss/RSSMarketsMain.xml')}"`**

   This change is necessary to remove double-encoding behavior, which requires manual correction.

1. Switch back to the designer.

```
